src/components/Contentful/DynamicHero.vue

```diff
@@ -173,6 +173,24 @@ export default {
 				link: this.heroText?.primaryCtaLink ?? '',
 			};
 		},
+		heroBackground() {
+			return this.content?.contents?.find(({ key }) => key.indexOf('background') > -1);
+		},
+		heroBackgroundStyle() {
```

Comment:
I think we should make these 2 properties utility methods that can be called from computed properties in other components as well.

Something along these lines...

contentGroupBackground(contents)
// `contents` can be termed differently across contexts outside of these landing pages.
+
contentGroupBackgroundStyles(backgroundContent)